core/src/main/resources/hudson/triggers/SCMTrigger/SCMAction/index_zh_CN.properties

```diff
@@ -1,3 +1,3 @@
 # This file is under the MIT License by authors
 
-title=9999999999999
```

Comment:
Hmm, sounds like a bug in the original translation

Reply:
Agree